test/extends.test.js

```diff
@@ -17,7 +17,7 @@ describe("extend", function() {
       result.should.have.type("string");
 
       // verify the generated module imports the `include`d templates
-      result.should.match(/require\(\"twig\!a\.html\.twig\"\);/);
```

Comment:
I don't think removing the twig loader prefix is a good idea. The webpack user would be forced to use a config file to map twig files to the twig loader, which is never the case in other loaders.

It shouldn't be too hard adding this to the included templates after processing.